homeassistant/components/tellduslive/__init__.py

```diff
@@ -72,16 +71,13 @@
     hass.data[DATA_CONFIG_ENTRY_LOCK] = asyncio.Lock()
     hass.data[CONFIG_ENTRY_IS_SETUP] = set()
 
-    client = TelldusLiveClient(hass, entry, session)
+    interval = entry.data[KEY_SCAN_INTERVAL]
+    _LOGGER.debug('Update interval %s seconds.', interval)
+    client = TelldusLiveClient(hass, entry, session, interval)
     hass.data[DOMAIN] = client
     await async_add_hubs(hass, client, entry.entry_id)
     hass.async_create_task(client.update())
```

Comment:
@MartinHjelmare is this what you meant? 

```suggestion
     hass.loop.create_task(client.update())
```